src/appComponents/common/ApiJsonFactory.js

```diff
@@ -1,29 +1,38 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
+(function () {
 
-var commonFactories = angular.module('openevent');
+"use strict";
+const commonFactories = angular.module("openevent");
+
+commonFactories.factory("ApiJsonFactory", ["$q", "$http", function ($q, $http) {
+
+const baseUrl = `(config.use_testApi ? "testapi/" : config.apiBaseGetUrl) event/ ${config.eventId}`;
 
-commonFactories.factory('ApiJsonFactory', ['$q', '$http', function ($q, $http) {
-var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
-    'event/' + config.eventId;
         return {
             getJson: function ($apiEndpoint) {
-                let endpoint;
-                if (($apiEndpoint === 'event') && (!config.use_testApi)) {
-                    endpoint = baseUrl;
+                let endpoint ="";
+                if (($apiEndpoint === "event") && (!config.use_testApi)) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier '$apiEndpoint' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-webapp/file/2432887269/issues/source?bid=3233857&fileBranchId=3238420#l16)